internal/service/project/resource_project.go

```diff
@@ -693,21 +693,21 @@ func UpdateProjectLimits(ctx context.Context, client GroupProjectService, projec
 	return nil
 }
 
-func setProjectLimits(ctx context.Context, client GroupProjectService, projectID string, tfLimits []TFLimitModel) error {
+func setProjectLimits(ctx context.Context, client *admin.APIClient, projectID string, tfLimits []TFLimitModel) error {
 	for _, limit := range tfLimits {
 		dataFederationLimit := &admin.DataFederationLimit{
 			Name:  limit.Name.ValueString(),
 			Value: limit.Value.ValueInt64(),
 		}
-		_, _, err := client.SetProjectLimit(ctx, limit.Name.ValueString(), projectID, dataFederationLimit)
+		_, _, err := client.ProjectsApi.SetProjectLimit(ctx, limit.Name.ValueString(), projectID, dataFederationLimit).Execute()
 		if err != nil {
 			return fmt.Errorf("error adding limits into the project: %v", err.Error())
 		}
 	}
 	return nil
 }
 
-func UpdateProjectTeams(ctx context.Context, client GroupProjectService, projectState, projectPlan *TFProjectRSModel) error {
+func UpdateProjectTeams(ctx context.Context, client *admin.APIClient, projectState, projectPlan *TFProjectRSModel) error {
```

Comment:
same

Reply:
Changed in [b781b71](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2045/commits/b781b71f45f8db1b5555c70847644d57fc30763f)